src/root_files/_common/404.html

```diff
@@ -40,6 +40,13 @@
     function redirect(lang) {
         var base = /^(.+?).github.io$/.test(window.location.hostname) ? window.location.origin + '/binary-static' : window.location.origin;
 
+        // TODO: REMOVE/CHANGE this if statement after addition of DE or TH languages
+        if (lang.toLowerCase() === 'de' || lang.toLowerCase() === 'th') {
```

Comment:
maybe instead of checking unsupported languages we can check supported languages?

Reply:
or if we want to keep this as checking unsupported ones, it could be an array and check indexof instead of individually checking each of them.

Reply:
@negarn As we talked before somewhere else the default language is coming from backend and I have to check for unsupported ones. I will make changes for array checking